orchestration/dagster_orchestration/hca_orchestration/tests/environments/test_load_hca_local_e2e.yaml

```diff
@@ -1,18 +1,24 @@
 resources:
   beam_runner:
     config:
-      target_class: "hca-transformation-pipeline"
+      target_class: hca-transformation-pipeline
   load_tag:
     config:
-      load_tag_prefix: "fake_load_tag_prefix"
       append_timestamp: true
+      load_tag_prefix: test_load_hca_local_e2e
   scratch_config:
     config:
-      scratch_bucket_name: broad-dsp-monster-hca-dev-temp-storage
       scratch_bq_project: broad-dsp-monster-hca-dev
-      scratch_dataset_prefix: local_e2e
+      scratch_bucket_name: broad-dsp-monster-hca-dev-temp-storage
+      scratch_dataset_prefix: test_load_hca_local_e2e
       scratch_table_expiration_ms: 3600000
+  target_hca_dataset:
+    config:
+      dataset_name: hca_dev_20210405_arh_testing
+      dataset_id: f413bcb3-9ec4-41b0-a75b-1e36b19dc3dc
+      project_id: broad-jade-dev-data
+      billing_profile_id: 390e7a85-d47f-4531-b612-165fc977d3bd
```

Comment:
Once we figure out the issue I mentioned in the PR description, I'm hopeful we can use this config to run e2e locally.